tsconfig.json

```diff
@@ -15,6 +15,12 @@
     "jsx": "preserve",
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "features/home/components/News.tsx",
+    "features/home/components/ClubsOverview.tsx"
```

Comment:
Not necessary. `**/*.tsx` is enough for compiling / transpiling all '.tsx' files...
```suggestion
```

Reply:
Changed by mistake!